### PR TITLE
Dev tools: Tidying up the wallet section

### PR DESCRIPTION
**Pull request by kmurphypolygon**
Gonna go through these open tickets in the backlog as soon as I have time to do them.

https://polygon.atlassian.net/browse/DOCS-189

### CODE REVIEW

File: docs/tools/wallets/portal.md
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md:1]: Polygon "Portal" is always mentioned with a capital "P" in blog posts and on the official site. 
See:
- https://polygon.technology/blog/introducing-polygon-portal-step-into-the-future-of-ux-in-web3
- https://coincodex.com/article/35542/polygon-portal-goes-live-streamlining-how-users-interact-with-the-web3-ecosystem/
- https://www.cointribune.com/en/crypto-polygon-launches-portal-a-bright-future-for-matic/
- https://polygon.technology/

Best to use the same wording here.

File: docs/tools/wallets/portal.md [hunk 1]
```diff
@@ -2,144 +2,173 @@
...
-
     If you come across any issues, [Polygon support](https://support.polygon.technology/support/home) is just a click away! Feel free to get in touch with us for assistance.
 
-## Connect to Polygon Portal
+## Connect to Polygon portal
 
-1. Use the following URL to access Polygon Portal: https://portal.polygon.technology/
+1. Use the following URL to access Polygon portal: https://portal.polygon.technology/
 
 2. Click on the "Connect Wallet" button to bring up the **blocknative** wallet connect window.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md]:
```suggestion
2. Click on the **Connect Wallet** button to bring up the **blocknative** wallet connect window.
```

File: docs/tools/wallets/portal.md [hunk 2]
```diff
@@ -2,144 +2,173 @@
...
 
-    ![](../../img/tools/portal/portal-25.png)
+    <center>
+    ![](../../img/tools/portal/portal-25.png){ width=60% }
+    </center>
 
 ## Transfer funds from Ethereum to Polygon chains
 
-1. Start by navigating to the [Polygon Portal](https://portal.polygon.technology/), and then either selecting "Bridge you Asset" option on the main page, or the "Bridge" option from the sidebar on your dashboard.
+1. Start by navigating to the [Polygon portal](https://portal.polygon.technology/), and then either selecting "Bridge you Asset" option on the main page, or the "Bridge" option from the sidebar on your dashboard.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md]:
```suggestion
1. Start by navigating to the [Polygon portal](https://portal.polygon.technology/), and then either selecting **Bridge your Asset** option on the main page, or the **Bridge** option from the sidebar on your dashboard.
```

File: docs/tools/wallets/portal.md [hunk 3]
```diff
@@ -2,144 +2,173 @@
...
+        You'll also see the option to transfer the tokens to a different address that you can select to enter a wallet address different from the one connected to the portal.
 
 5. At this point, you should be able to send the transaction through. Confirm the transaction in your wallet, and you'll be able to see an ETA for the funds to hit your wallet on the destination chain.
 
-    ![](../../img/tools/portal/portal-7.png)
+    <center>
+    ![](../../img/tools/portal/portal-7.png){ width=60% }
+    </center>
 
 6. You can check the transaction status and browse your transaction history by selecting "Transactions" from the sidebar. Select a transaction to view details.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md]:
```suggestion
6. You can check the transaction status and browse your transaction history by selecting **Transactions** from the sidebar. Select a transaction to view details.
```

File: docs/tools/wallets/portal.md [hunk 4]
```diff
@@ -2,144 +2,173 @@
...
 
-    ![](../../img/tools/portal/portal-9.png)
+    ![](../../img/tools/portal/portal-9.png){ width=60% }
+    </center>
 
 ## Claim bridged tokens on Ethereum
 
 ### MATIC tokens
 
 1. Let's say you bridged MATIC from PoS to Ethereum. Once your tokens are ready to claim, you'll be able to select the "Initiate Claim" option under the transaction record.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md]:
```suggestion
1. Let's say you bridged MATIC from PoS to Ethereum. Once your tokens are ready to claim, you'll be able to select the **Initiate Claim** option under the transaction record.
```

File: docs/tools/wallets/portal.md [hunk 5]
```diff
@@ -2,144 +2,173 @@
...
 
-    ![](../../img/tools/portal/portal-14.png)
+    <center>
+    ![](../../img/tools/portal/portal-14.png){ width=60% }
+    </center>
 
 2. Approve the transaction in your wallet, and then select "Claim Tokens".
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md:110]:
```suggestion
2. Approve the transaction in your wallet, and then select **Claim Tokens**.
```

File: docs/tools/wallets/portal.md [hunk 6]
```diff
@@ -2,144 +2,173 @@
...
-    ![](../../img/tools/portal/portal-15.png)
+    <center>
+    ![](../../img/tools/portal/portal-15.png){ width=60% }
+    </center>
 
 3. The funds will be reflected in your wallet soon. You can keep an eye on the transaction status from this page.
 
 ### ETH and custom tokens
 
 1. Your tokens transferred from Polygon zkEVM to Ethereum will generally be available to claim in 30-45 minutes. You can track the transaction status on the "Transactions" page.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md:120]:
```suggestion
1. Your tokens transferred from Polygon zkEVM to Ethereum will generally be available to claim in 30-45 minutes. You can track the transaction status on the **Transactions** page.
```

File: docs/tools/wallets/portal.md [hunk 7]
```diff
@@ -2,144 +2,173 @@
...
 
-    ![](../../img/tools/portal/portal-10.png)
+    <center>
+    ![](../../img/tools/portal/portal-10.png){ width=60% }
+    </center>
 
 2. The "Claim Tokens" button will appear under the transaction once your tokens are ready to be claimed.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md]:
```suggestion
2. The **Claim Tokens** button will appear under the transaction once your tokens are ready to be claimed.
```

File: docs/tools/wallets/portal.md [hunk 8]
```diff
@@ -2,144 +2,173 @@
...
 
-    ![](../../img/tools/portal/portal-13.png)
+    <center>
+    ![](../../img/tools/portal/portal-13.png){ width=60% }
+    </center>
 
-## Add custom tokens to Polygon Portal
+## Add custom tokens to Polygon portal
 
 1. First, try looking up the token directly by clicking on the "Manage Tokens" located in the bottom-right corner of the dashboard, and then entering the token name in the input field.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md]:
```suggestion
1. First, try looking up the token directly by clicking on the **Manage Tokens** located in the bottom-right corner of the dashboard, and then entering the token name in the input field.
```

File: docs/tools/wallets/portal.md [hunk 9]
```diff
@@ -2,144 +2,173 @@
...
 
-    ![](../../img/tools/portal/portal-16.png)
+    <center>
+    ![](../../img/tools/portal/portal-16.png){ width=60% }
 
-    ![](../../img/tools/portal/portal-26.png)
+    ![](../../img/tools/portal/portal-26.png){ width=60% }
+    </center>
 
 2. If the token you're looking for shows up, select "Add" to add it to the portal.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md]:
```suggestion
2. If the token you're looking for shows up, select **Add** to add it to the portal.
```

File: docs/tools/wallets/portal.md [hunk 10]
```diff
@@ -2,144 +2,173 @@
...
 
 3. If your token doesn't appear in the search results, you can also add the token manually by selecting the "+" in the bottom-right of the "Manage tokens" window.
```
Comment (hsutaiyu) [inline docs/tools/wallets/portal.md]:
```suggestion
3. If your token doesn't appear in the search results, you can also add the token manually by selecting the **+** in the bottom-right of the **Manage tokens** window.
```

File: mkdocs.yml [hunk 11]
```diff
@@ -492,27 +493,25 @@ nav:
           - Umbrella: tools/oracles/umbrella.md
       - Wallets:
           - Getting started:  tools/wallets/getting-started.md
-          - Polygon Portal: tools/wallets/portal.md
+          - Polygon portal: tools/wallets/portal.md
```
Comment (hsutaiyu) [inline mkdocs.yml]: See comment above regarding capitalizing the "P" in "Portal".

**Review by hsutaiyu — APPROVED** [review]
Approved with fixes

File: mkdocs.yml (hunk 11 above)
Comment (kmurphypolygon) [reply mkdocs.yml]: yeah, that is true ... i was looking on the portal itself ... which has lower case polygon too :D let me update

**Pull request merged**